.github/workflows/release.yml

```diff
@@ -94,26 +96,39 @@ jobs:
             tar czvf ../../../${{ matrix.platform.name }} ${{ matrix.platform.bin }}
           fi
           cd -
-        if: |
-          matrix.toolchain == 'stable' &&
-          ( startsWith( github.ref, 'refs/tags/v' ) ||
-            github.ref == 'refs/tags/test-release' )
+        if: (startsWith(github.ref, 'refs/tags/test-release') || startsWith(github.ref,'refs/tags/v'))
+      - name: Generate SHA-256
+        run: shasum -a 256 ${{ matrix.platform.name }}
+        if: (startsWith(github.ref, 'refs/tags/test-release') || startsWith(github.ref,'refs/tags/v'))&&matrix.platform.os == 'macOS-latest'
```

Comment:
why do we need `matrix.platform.os == 'macOS-latest'`?

Reply:
I didn't modify this condition, just kept as it was. I considered it as a ready for homebrew, which use it to validation. Anyway, sha256 is also important for anyone to validate a manually downloaded packages, so that condition can/should be removed, I think.

Reply:
Yeah, I just checked it in the original  code, It seems I forgot to remove the debug code before. You can remove it now I think.